host/src/containers/GameSessionContainer.tsx

```diff
@@ -187,33 +232,40 @@ const GameSessionContainer = () => {
   const handleBeginQuestion = () => {
     // I'm keeping this console.log in until we figure out NOT_STARTED so we can tell there's been a change in state 
     console.log(gameSession.currentState);
+    const gameSessionId = gameSession?.id;
+    const questionId = gameSession.questions[gameSession.currentQuestionIndex].id;
+    const order = gameSession.questions[gameSession.currentQuestionIndex].order;
     if (gameSession.currentState === GameSessionState.TEAMS_JOINING) {
-      if (isConfidenceEnabled === true ){
         apiClient.updateQuestion({ gameSessionId: gameSessionId, id: gameSession.questions[0].id, order: 0, isConfidenceEnabled: isConfidenceEnabled })
           .then(response => {
-            console.log(response);
-          });
-      }
-
-      let newUpdates = { currentState: GameSessionState.CHOOSE_CORRECT_ANSWER };
-      apiClient.updateGameSession({ id: gameSessionId, ...newUpdates })
-        .then(response => {
-          localStorage.setItem('currentGameTimeStore', gameSession.phaseOneTime);
-          setHeaderGameCurrentTime(gameSession.phaseOneTime);
-          checkGameTimer(response);
-          setGameSession(response);
-          const teamDataRequests = response.teams.map(team => {
-            return apiClient.getTeam(team.id); // got to call the get the teams from the API so we can see the answers
-          });
-
-          Promise.all(teamDataRequests)
-            .then(responses => {
-              setTeamsArray(responses);
-            })
-            .catch(reason => console.log(reason));
-        });
-      setIsTimerActive(true);
-      setIsLoadModalOpen(true);
+            if (gameSession.currentState === GameSessionState.TEAMS_JOINING) {
```

Comment:
Maybe do an early exist here by changing it to `!==` and return. That helps the rest of the code below to be indented one less